content/anchor-cpi.md

```diff
@@ -392,6 +392,11 @@ Next, let’s update the `add_movie_review` instruction to do the following:
 
 Fortunately, we can use the `anchor_spl` crate to access helper functions and types like `mint_to` and `MintTo` for constructing our CPI to the Token Program. `mint_to` takes a `CpiContext` and integer as arguments, where the integer represents the number of tokens to mint. `MintTo` can be used for the list of accounts that the mint instruction needs.
 
+```rust
+use anchor_spl::token::{mint_to, MintTo, Mint, TokenAccount, Token};
+use anchor_spl::associated_token::AssociatedToken;
+```
+
```

Comment:
Why the two code blocks, one straight after the other? 

I *think* this should be something like:

Update your `use` statements to include:

```
use anchor_spl::token::{mint_to, MintTo, Mint, TokenAccount, Token};
use anchor_spl::associated_token::AssociatedToken;
```

Then add the instruction handler:

```
(instruction handler code)
```